src/go/agwd/server/server.go

```diff
@@ -127,8 +137,32 @@ func startSctpdUplinkServer(
 	go grpcServer.Serve(listener)
 }
 
+func startConfigServer(
+	cfgr config.Configer, logger log.Logger,
+) {
+	target := config.ParseTarget(cfgr.Config().GetConfigServiceTarget())
+	if target.Scheme == "unix" {
+		cleanupUnixSocketOrDie(logger, target.Endpoint)
+	}
+
+	listener, err := net.Listen(target.Scheme, target.Endpoint)
+	if err != nil {
```

Comment:
1. Hmmm I wonder if this could be DRY'd up
2. Multiple services can be configured with the same target. we should update/refactor this code so that the startup code creates listeners for each unique target then maps services to those listeners. this can be in a followup

Reply:
Agreed we can clean this up, ill add a follow on task for that to avoid adding to much to this pr. #10131